### ISSUE TITLE
wht change ios macro 1 to 0

### ISSUE THREAD

**Issue by jinliyuelong** [Baseflow/flutter-permission-handler#650]
My previous version was 5.1. I saw that the macro defined in IOS source code was 1, but after upgrading to 8.1, I found that the defined macro became 0, resulting in my failure to obtain permission. Why should I modify the meaning of macros

![image](https://user-images.githubusercontent.com/16810284/129661228-dad56e61-a025-41d5-b002-c496d977c57e.png)

such as 
#if PERMISSION_PHOTOS

#import <Photos/Photos.h>

@interface PhotoPermissionStrategy : NSObject <PermissionStrategy>
-(instancetype)initWithAccessAddOnly:(BOOL) addOnly;
@end

#else

#import "UnknownPermissionStrategy.h"
@interface PhotoPermissionStrategy : UnknownPermissionStrategy
@end

#endif

it goes  PhotoPermissionStrategy with UnknownPermissionStrategy.so that it can not work

**Comment by mvanbeusekom** [comment]
We flipped the Macro definitions to support adding new permissions without having to do a breaking change release.

In earlier versions all permissions were enabled by default meaning if we introduce support for a new permission, developers were forced to update their `ios/Podfile` and disable the new permission if they didn't need to support it. 

Since version 8.0 we changed this behavior and treat permissions default as disabled (causing no breaking changes for developers when we introduce a new permission).

The behavior has been documented in our README (see the iOS section) and the CHANGELOG.
